tests/integration_tests.rs

```diff
@@ -622,6 +622,58 @@ fn complex_parallel_for_appender_loop() {
     unsafe { weld_value_free(ret_value) };
 }
 
+fn simple_for_vectorizable_loop() {
+    #[allow(dead_code)]
+    struct Args {
+        x: WeldVec<i32>,
+    }
+
+    let code = "|x:vec[i32]| result(for(x, merger[i32,+], |b,i,e| let a = 1; let a2 = a + 1; merge(b, e+a2)))";
+    let conf = default_conf();
+
+    let size = 1000;
```

Comment:
Can you make the sizes of this and other vectorization tests not multiples of 4 so that fringing is tested?